gtest-parallel

```diff
@@ -266,12 +266,10 @@ parser.add_option('--format', type='string', default='filter',
                   help='output format (raw,filter)')
 parser.add_option('--print_test_times', action='store_true', default=False,
                   help='When done, list the run time of each test')
-parser.add_option('--shard_count', type='int',
-                  default=int(os.environ.get('GTEST_TOTAL_SHARDS', 1)),
+parser.add_option('--shard_count', type='int', default=1,
                   help=('Total number of shards (for sharding test execution '
```

Comment:
lowercase help, remove ()s around help and remove default values + "."s

e.g. help='total number of shards (for sharding test execution between multiple machines)'

And

help='zero-indexed number identifying this shard (for sh..'

Can you also change the above help from "When done.." to 'list the run time of each test at the end of execution' ?